ignition-transport6.rb

```diff
@@ -0,0 +1,57 @@
+class IgnitionTransport6 < Formula
+  desc "Transport middleware for robotics"
+  homepage "https://ignitionrobotics.org"
+  url "https://bitbucket.org/ignitionrobotics/ign-transport/get/efa81ed82c53.tar.gz"
+
+  version "5.999.999~20180620~efa81ed"
+  sha256 "acfc4d4cd62eeb27371ac5ada1c596b5a3d7a221af4dad20121234540a5f9713"
+
+  depends_on "doxygen" => [:build, :optional]
+  depends_on "protobuf-c" => :build
+
+  depends_on "cmake"
+  depends_on "cppzmq"
+  depends_on "ignition-cmake1"
```

Comment:
Should ign-transport6 use `ignition-cmake2`?

Reply:
Changed.